### ISSUE TITLE
contact form validates but doesn't send emails to domain server email. How can I fix? Thank you for your feedback

### ISSUE THREAD

**Issue by maiixone** [maiixone/maiix#1]
<?php
session_start();
if(!$_SESSION["a"]){
    $a = rand(1,50);
    $b = rand(1,50);
     $_SESSION["a"] = $a;
    $_SESSION["b"] = $b;  
}
$email = '';
$message = '';
$name ='';
$result ='';
$error ='';
if(isset($_POST['submit'])){
      // echo '<h1>Submit Clicked !!!! '.$_POST['submit'].'</h1>';
        $name = $_POST['name'];
        $email = $_POST['email'];
        $message = $_POST['message'];
        $check = intval($_POST['secCheck']);
    
        if(!$_POST['name']){ $error .= 'Please enter your name! <br>'; }
        if(!$_POST['email'] || !filter_var($_POST['email'], FILTER_VALIDATE_EMAIL)){ $error .= 'Please enter your email! <br>'; }
        if(!$_POST['message']){ $error .= 'Please enter a message! <br>'; }
        if($check !== ($_SESSION["a"] + $_SESSION["b"])){ $error .= 'Wrong AntiSpam value <br>'; }  
 
if($error == ''){
    $from = 'maiix@mydomainemail.com';
    $to =   'maiix@dmydomainemail.com'; // email that you want to receive the email to. YOUR ADDRESS
    $subject = 'Message from contact form';
    $body = "From: $name ($email) \n Message \n $message";
    $headers = "MIME-Version: 1.0" . "\r\n";
    $headers .= "Content-type:text/html;charset=UTF-8" . "\r\n";
    $headers .= 'From: <'.$from.'>' . "\r\n";  
    if(@mail($to,$subject,$body,$headers)){
     $result = '<div class="alert alert-success">Mail Sent</div>';  
        }else{
               $result = '<div class="alert alert-success">Mail wasn\'t sent</div>';    
        }
    $email = '';
    $message = '';
    $name ='';
    
    // Send email
} else {
    $result = '<div class="alert alert-danger">Error Found:<br>'.$error.'</div>';
}
    }
?>
	.text-danger{
	display:none;
	}
	</style>
</head>

<body>
	  <br>
<br>
	<style>
    .text-danger {
        display:none;
     }
 
     </style>
    </head>
    <body>
   <div class="container">
   <div class="row">
	<div class="col-sm-10 text-left">   
   <form method="post" action="contactpge.php" class="form-horizontal">
   <div class="form-group">
   <label for="name" class="col-sm-2 control-label"><h4>Name</h4></label>
   <div class="col-sm-10">
   <input type="text" id="name" name="name" placeholder="Enter your name" class="form-control" value="<?php echo $name;?>">
   <p class="text-danger">You need to enter a name value</p>
    </div>
    </div>
   <div class="form-group">
  <label for="email" class="col-sm-2 control-label"><h4>Email</h4></label>
   <div class="col-sm-10">
  <input type="email" id="email" name="email" placeholder="your@email.com" class="form-control" value="<?php echo $email;?>">
  <p class="text-danger">You need to enter a valid email</p>
   </div>
    </div>
   <div class="form-group">
   <label for="message" class="col-sm-2 control-label"><h4>Message</h4></label>
   <div class="col-sm-10">
	<textarea id="message" name="message" rows="4" class="form-control"><?php echo $message;?></textarea>
   <p class="text-danger">You need to enter a message</p>
   </div>
   </div>
   <div class="form-group">
    <label for="secCheck" class="col-sm-2 control-label">
     <?php echo $_SESSION["a"]  . ' + ' .$_SESSION["b"] ; ?>
      </label>
      <div class="col-sm-10">
    <input type="text" id="secCheck" name="secCheck" class="form-control">
     <p class="text-danger">Answer the question above</p>
     </div>
      </div>
       <div class="form-group">
      <div class="col-sm-10 col-sm-offset-2">
      <input type="submit" value="Send" class="btn btn-primary btn-large btn-block" id="submit" name="submit">
       </div>
       </div>
       <div class="form-group">
         <div class="col-sm-10 col-sm-offset-2">
         <?php echo $result; ?>
          </div>
           </div>
          </form>
      </div>
    </div>
	   </div>
	<br>
      <script src="http://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
	<script src="js/jquery-1.11.3.min.js"></script>
	 
	<script src="js/bootstrap.js"></script>
  <script>
            $('form').on('submit',function(){
                $(".text-danger").hide();
                var holderValue = false;
                $(this).find('input[type!="hidden"]').each(function(){
                    if(!$(this).val()){holderValue =true; $(this).parent().find(".text-danger").show();}
                })
             if(!$("#message").val()  ){holderValue =true; $("#message").parent().find(".text-danger").show();};
     
                //event.preventDefault();
               /// console.log($('#message').val() );
                
            })
        </script>
</body>
</html>
